aws-rds-customdbengineversion/src/main/java/software/amazon/rds/customdbengineversion/Translator.java

```diff
@@ -31,9 +31,12 @@ public static CreateCustomDbEngineVersionRequest createCustomDbEngineVersionRequ
                 .description(model.getDescription())
                 .engine(model.getEngine())
                 .engineVersion(model.getEngineVersion())
+                .imageId(model.getImageId())
                 .kmsKeyId(model.getKMSKeyId())
                 .manifest(model.getManifest())
+                .sourceCustomDbEngineVersionIdentifier(model.getSourceCustomDBEngineVersionIdentifier())
```

Comment:
Could you please verify if there have been any recent additions to the CEV statuses for restoration, such as the "restoring" status? If there is a new status, it will need to be added to the followingenum: https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-rds/blob/master/aws-rds-customdbengineversion/src/main/java/software/amazon/rds/customdbengineversion/CustomDBEngineVersionStatus.java . Additionally, please ensure that the new status is marked as "Transient" to ensure successful stabilization logic.